src/schema/notifications.ts

```diff
@@ -145,6 +145,26 @@ export class ZimbraNotifications {
 		this.getApolloClient().queryManager.broadcastQueries();
 	};
 
+	// Find the actual folder of the shared folder
+	private findSharedItemId = (item: any) => {
```

Comment:
Can we destructure id here?

Reply:
Tweaked logic to only send id to `findSharedItemId`, hence destructuring will not be required.